packages/core/src/molecule/codec.ts

```diff
@@ -43,7 +49,7 @@ export class Codec<Encodable, Decoded = Encodable> {
         }
         return encoded;
       },
-      (decodable) => {
+      (decodable, config = { isExtraFieldIgnored: false }) => {
```

Comment:
I recommend not setting the default value here. Just leave it undefined. Adding more config fields in the future and forgetting to modify every default value could cause a problem.